DigitalLearningSolutions.Data/Services/SectionContentService.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System.Data;
+    using System.Linq;
+    using Dapper;
+    using DigitalLearningSolutions.Data.Models.SectionContent;
+    using Microsoft.Extensions.Logging;
+
+    public interface ISectionContentService
+    {
+        SectionContent? GetSectionContent(int customisationId, int candidateId, int sectionId);
+    }
+
+    public class SectionContentService : ISectionContentService
+    {
+        private readonly IDbConnection connection;
+        private readonly ILogger<SectionContentService> logger;
+
+        public SectionContentService(IDbConnection connection, ILogger<SectionContentService> logger)
+        {
+            this.connection = connection;
+            this.logger = logger;
+        }
+
+        public SectionContent? GetSectionContent(int customisationId, int candidateId, int sectionId)
+        {
+            return connection.QueryFirstOrDefault<SectionContent>(
+                @"
+                    SELECT
+                        Customisations.CustomisationName,
+                        Applications.ApplicationName,
+                        Sections.SectionName,
+                        SUM(aspProgress.TutTime) AS TimeMins, 
+                        Sections.AverageSectionMins AS AverageSectionTime, 
+                        dbo.CheckCustomisationSectionHasLearning(Customisations.CustomisationID, Sections.SectionID) AS HasLearning,
+                        (CASE
+                            WHEN Progress.CandidateID IS NULL
+                                OR dbo.CheckCustomisationSectionHasLearning(Progress.CustomisationID, Sections.SectionID) = 0
+                            THEN 0
+                            ELSE CAST(SUM(aspProgress.TutStat) * 100 AS FLOAT) / (COUNT(Tutorials.TutorialID) * 2)
+                        END) AS PercentComplete
+                    FROM
+                        Tutorials
+                        INNER JOIN CustomisationTutorials
+                            ON CustomisationTutorials.TutorialID = Tutorials.TutorialID
+                        INNER JOIN Customisations
+                            ON Customisations.CustomisationID = CustomisationTutorials.CustomisationID
+                        INNER JOIN Applications
+                            ON Applications.ApplicationID = Customisations.ApplicationID
+                        INNER JOIN Sections
+                            ON Sections.SectionID = Tutorials.SectionID
+                        LEFT JOIN Progress
+                            ON Progress.CustomisationID = Customisations.CustomisationID
+                            AND Progress.CandidateID = @candidateId
```

Comment:
Should also check here that removed date is null and system refreshed is 0, as in the query for the initial menu 🙂